scripts/lib/main_test.go

```diff
@@ -0,0 +1,81 @@
+package slacklog_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"testing"
+)
+
+func cleanupTmpDir(t *testing.T, path string) {
+	t.Helper()
+
+	err := os.RemoveAll(path)
+	if err != nil {
+		t.Fatalf("failed to cleanupTmpDir: %s", err)
+	}
+}
+
+func createTmpDir(t *testing.T) string {
+	t.Helper()
+
+	path, err := ioutil.TempDir("testdata", "slacklog")
```

Comment:
```suggestion
	path, err := ioutil.TempDir("", "slacklog")
```

`""`  ならばOSが用意してくれるtmpディレクトリを使ってくれ、そちらのほうが良いです。
仮に testdata/ 下にしなければならない理由があるならそれをコメントしたほうが良いです。
できるだけ testdata/ 下には書きたくないので。

Reply:
#44 で修正